frontend/src/layout/header/UserBox.tsx

```diff
@@ -33,71 +33,135 @@ export const UserBox: React.FC = () => {
-            <Link
-                to={CONFIG.auth.loginLink ?? LOGIN_PATH}
-                onClick={() => {
-                    // Store a redirect link in session storage.
-                    window.sessionStorage.setItem(REDIRECT_STORAGE_KEY, window.location.href);
-                }}
-                htmlLink={!!CONFIG.auth.loginLink}
-                css={{
-                    alignSelf: "center",
-                    borderRadius: 10,
-                    cursor: "pointer",
-                    padding: "5px 14px",
-                    marginRight: 8,
-                    display: "flex",
-                    gap: 8,
-                    alignItems: "center",
-                    backgroundColor: "var(--nav-color)",
-                    color: "var(--nav-color-bw-contrast)",
-                    "&:hover, &:focus": {
+    return <>
+        <Link
+            to={CONFIG.auth.loginLink ?? LOGIN_PATH}
+            onClick={() => {
+                // Store a redirect link in session storage.
+                window.sessionStorage.setItem(REDIRECT_STORAGE_KEY, window.location.href);
+            }}
+            htmlLink={!!CONFIG.auth.loginLink}
+            css={{
+                outline: "transparent",
+                ":hover, :focus": {
+                    "> :first-child": {
                         backgroundColor: "var(--nav-color-dark)",
                         color: "var(--nav-color-bw-contrast)",
                     },
-                    "&:focus-visible": {
-                        outline: "none",
-                        boxShadow: "0 0 0 2px black",
-                    },
-                    [`@media (max-width: ${BREAKPOINT_MEDIUM}px)`]: {
+                },
+                ":hover > div": {
+                    opacity: 1,
+                    outline: "2px solid var(--grey80)",
+                },
+                ":focus > div": {
+                    opacity: 1,
+                    outline: "2px solid var(--accent-color)",
+                },
+            }}
+        >
+            <div css={{
+                color: "var(--nav-color-bw-contrast)",
+                display: "flex",
+                alignItems: "center",
+                gap: 8,
+                borderRadius: 8,
+                padding: "7px 14px",
+                backgroundColor: "var(--nav-color)",
+                outlineOffset: 1,
+                [`@media (max-width: ${BREAKPOINT_MEDIUM}px)`]: {
+                    display: "none",
+                },
+            }}>
+                <FiLogOut size={"20px"} />
+                {t("user.login")}
+            </div>
+            {/* Show icon on mobile devices. */}
+            <div css={{
+                color: "black",
+                ...ICON_STYLE,
+                [`@media not all and (max-width: ${BREAKPOINT_MEDIUM}px)`]: {
+                    display: "none",
+                },
+            }}>
+                <FiLogOut />
```

Comment:
Why are we using the logout icon for this login button? :thinking:

Reply:
Again, suggested by the redesign. I think we talked about this like a month ago and you didn't mind, but I'm also slighly confused by that change.

Reply:
Ok so here are some screenshots:
This is the logout mirrored, which I think looks kinda weird. Not really a fan.
<img width="292" alt="logout_mirrored" src="https://user-images.githubusercontent.com/94838646/221642068-e733fe76-e3df-4d9f-92c7-6bb8ca37e1fe.png">
Login mirrored. Somehow also weird. Might be a left-to-right cultural thing or sth though.
<img width="292" alt="login_mirrored" src="https://user-images.githubusercontent.com/94838646/221642329-894d61bd-d94c-4f74-b44c-ac7f592e2ecd.png">
Regular login. This is the one that was used before this PR and also my favorite. Maybe with slightly less padding on the left or a larger gap between icon and text 🤷 
<img width="291" alt="login_regular" src="https://user-images.githubusercontent.com/94838646/221642221-bdd47b88-b428-499a-b317-11b7607fc53b.png">

Reply:
Jep, I agree: the normal login is my favorite.